.gitbook/5-oracles/1-operate-oracle-nodes/7-verify-remote-report.md

```diff
@@ -0,0 +1,92 @@
+# Verify Remote Report
+
+# Genesis Oracle
+
+- Status: Draft
+- Created: 2023-01-12
+- Modified: 2023-01-12
+- Authors
+    - Gyuguen Jang <gyuguen.jang@medibloc.org>
+    - Youngjoon Lee <yjlee@medibloc.org>
+    - Hansol Lee <hansol@medibloc.org>
+    - Myongsik Gong <myongsik_gong@medibloc.org>
+    - Inchul Song <icsong@medibloc.org>
+
+
+## Synopsis
+
+This document describes how to validate remote reports generated by Oracle.
+
+## Remote Report
+
+Remote reports are one of the core of our protocol. This remote report must be submitted when the genesis oracle or new oracle participant registers with Panacea. A remote report can verify that the genesis oracle or new oracle participant is the correct oracle. 
```

Comment:
How about some more specific explanation of what a correct oracle is? If you write a few lists with bullets, it seems to better express what a verify report means.